test_values.go

```diff
@@ -17,4 +19,16 @@ var testGateway = New(
 	os.Getenv("BRAINTREE_PRIV_KEY"),
 )
 
+var testTimeZone = func() *time.Location {
+	tzName := os.Getenv("BRAINTREE_TIMEZONE")
+	if tzName == "" {
+		tzName = "US/Mountain"
```

Comment:
Aha, I see! Then yes, I think defaulting to `UTC` and setting `BRAINTREE_TIMEZONE` in the `.travis.yml` makes sense.